app/js/showControllers.js

```diff
@@ -260,6 +273,16 @@ $stateParams, $state) {
 
 })
 
+
+// Link to the required Module
+.filter('linkModule', function($sanitize) {
+  return function(module) {
+    if (!module) return null
+
+      return 'https://modules.openmrs.org/#/show/' + module.slug
```

Comment:
Use `Config.appUrl` here to pull the URL from the config file (modulusui.conf.js). That way this directive doesn't break on non-production servers.